swagger.yaml

```diff
@@ -721,6 +721,30 @@ paths:
                 items:
                   $ref: "#/components/schemas/Error"
 
+  "/severa/flextime/{severaGuid}": 
+    get:
+      tags:
+        - UsersRead
+      summary: 
+        Get the flextime balance for a user. Total balance
+        is returned for the current date. Month balance is the balance for the
+        month of the current date.
+      operationId: getFlextimeByUser
```

Comment:
Looking at the ordering of the other endpoints in the spec its typically:
operationId
summary
description
tags
parameters
responses

Please reorder accordingly